data/assets/scene/solarsystem/planets/earth/transforms_GSE.asset

```diff
@@ -0,0 +1,51 @@
+local transforms = asset.require("scene/solarsystem/planets/earth/transforms")
+local coreKernels = asset.require("spice/core")
+local Eclipdate = asset.resource("kernels/ECLIPDATE.tf")
+local GSEKernel = asset.resource("kernels/GSE.tf")
+
+local Frame = {
+  GSE = "GSE"
+}
+local GSEReferenceFrame = {
+  Identifier = "GSEReferenceFrame",
```

Comment:
```suggestion
  Identifier = "GeocentricSolarEcliptic",
```

No need to say ReferenceFrame as it doesn't make a different to a user whether it is just a reference frame or if it has a renderable attached to it